test/core/end2end/grpc_core_end2end_test.bzl

```diff
@@ -166,7 +165,7 @@ _TESTS = [
     "write_buffering_at_end",
 ]
 
-def grpc_core_end2end_test_suite(name, config_src, deps = [], shard_count = 50, enable_fuzzing = True, tags = [], flaky = False):
+def grpc_core_end2end_test_suite(name, config_src, deps = [], shard_count = 50, enable_fuzzing = True, tags = [], flaky = False, with_no_logging_test = True):
```

Comment:
Are there files missing from this PR?  I'm assuming that in order to actually use this, we need something to call this function with the parameter set to false, right?

Reply:
For OSS we always want this true: it's important to have the no_logging tests.

I'm adding a parameter as a separate hedge for cases where (internally) we might have tests that don't work with the no_logging config.